hardwareModules/universalModule/software/esp32/dynamixelConfigurator/enumIntrospection.hpp

```diff
@@ -0,0 +1,243 @@
+#include <algorithm>
+#include <cctype>
+
+#include <peripherals/dynamixel.hpp>
+
+// Magic Enums are broken for GCC 8.4; let's make things simple, stupid...
+
+using namespace rofi::hal::dynamixel;
```

Comment:
I consider this as fine since it is not in the library but in the final binary that is a more-less quick & dirty tool. If we could script on ESP32, we would use scripting.